src/components/Buyer/AddPurchase.js

```diff
@@ -1,7 +1,40 @@
-import React from "react";
+import React from 'react'
+import { Link } from 'react-router-dom'
+import './AddPurchase.css'
 
 function AddPurchase() {
-  return <div>AddPurchase</div>;
+  return (<>
+  <p className="title">Purchase Carbon Credit</p>
+    <div className='signup-form2'>
+        <>
+        <p>Your have opted to purchase <Link className='linkstyle'>10,000 trees planted on 
+        2 acres of land. Offsetting 2T of Co2</Link> at <b>$10,000</b>. 
+        Complete the details below and Checkout to finish the 
+        transaction.
```

Comment:
-Fetch data from (`api/v1/carbon_credits/params`) so you can display data of the carbon credit to be bought instead of dummy data

Reply:
I have updated it as per the review check for further modification if necessary